csharp-basic-topics/ParamsKeyword/ParamsKeyword/CallParamsMethod.cs

```diff
@@ -0,0 +1,82 @@
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace ParamsKeyword
+{
+    [TestFixture]
+    public class CallParamsMethod
+    {
+        [Test]
+        public void CommaDelimitedListOfArgs_ReturnsValidResult()
```

Comment:
Please use the naming conventions from the guidelines so we can have a consistent experience throughout the code base.